document/FileFormat.mdk

```diff
@@ -913,16 +941,28 @@ An `invocation` object may contain a property named `commandLine` whose value is
 containing the runtime parameters with which the tool was invoked.
 For command line tools, this string may consist of the completely specified command line used to invoke the tool.
 
-~ IsoNote
+~ IsoNumberedNote { @IsoNumberedNote=1 }
 The information in the `commandLine` property makes it possible to precisely repeat a run of an analysis tool,
 and to verify that the results reported in the log file were generated by an appropriate invocation of the tool.
 ~
 
+The value of the `commandLine` property may be an empty string.
```

Comment:
Not if the tool is invoked in some way other than by a command line. I'll clarify.

---
In reply to: [75181477](https://github.com/sarif-standard/sarif-spec/pull/256#discussion_r75181477) [](ancestors = 75181477)